### ISSUE TITLE
missing relationship between cart and customer

### ISSUE THREAD

**Issue by shareef-dweikat** [zandoan/turing-frontend#9]
Using this API, i can generate carts. But there is no way to connect a cart generated with a customer!
I tried looking for such a feature in customer APIs as well. Unfortunately, could not find it either.
![Capture](https://user-images.githubusercontent.com/10628302/54707589-214f0280-4b4a-11e9-909e-250de0b63f94.PNG)

**Comment by azecharias** [comment]
You need to store the unique cart_id on the frontend for later use and always add items to a cart by using that cart_id

**Comment by shareef-dweikat** [comment]
@azecharias 
that is exactly what i did. the problem with this technic is that a cart will be stored locally on a device. so, you cant access it form another device.

**Comment by alejovdev** [comment]
Yes, it would be useful a post endpoint to the customer to set the cart id
